sdk/template/Azure.Template/api/Azure.Template.netstandard2.0.cs

```diff
@@ -33,3 +33,11 @@ public partial class SecretBundle
         public string Value { get { throw null; } }
     }
 }
+namespace Microsoft.Extensions.Azure
```

Comment:
Should this be suppressed also?

Reply:
This happens because the template gets generated and now contains the asp integrations.  I don't think we want to suppress this unless I am missing something